website/docs/reference/deploy/import-export.md

```diff
@@ -64,7 +64,9 @@ For example if you want to download just feature-toggles as yaml:
 
 ### API Import {#api-import}
 
-:::caution Importing environments
+:::caution Importing environments in Unleash 4.20 and below
+
+This is only relevant if you use **Unleash 4.20 or earlier**:
```

Comment:
@ivarconr We should make sure that the versions here are correct. If we just released 4.20, I can imagine this would be part of 4.21.